app/models/concerns/aggregatable.rb

```diff
@@ -35,8 +35,13 @@ class AggregateListError < StandardError; end
 
   included do
     belongs_to :game, touch: true
-    has_many :list_items, -> { index_order }, class_name: list_item_class_name, dependent: :destroy, foreign_key: :list_id
-    belongs_to :aggregate_list, class_name: to_s, foreign_key: :aggregate_list_id, optional: true
+    has_many :list_items,
+             -> { index_order },
+             class_name:  list_item_class_name,
+             dependent:   :destroy,
+             foreign_key: :list_id,
+             inverse_of:  :list
```

Comment:
It mainly wanted me to add the `:inverse_of` but that made the line really long so I reformatted it too.